frontend/src/components/organisms/ManageAttendees.tsx

```diff
@@ -125,6 +135,234 @@ const AttendeesTable = ({
   );
 };
 
+interface modalProps {
+  handleClose: () => void;
+  eventDetails?: FormValues;
+  eventid: string;
+}
+
+type FormValues = {
+  startDate: string;
+  endDate: string;
+  startTime: string;
+  endTime: string;
+};
+
+/** A duplicate event modal body */
+const ModalBody = ({ handleClose, eventDetails, eventid }: modalProps) => {
+  const { user } = useAuth();
+  const queryClient = useQueryClient();
+
+  /** State variables for the notification popups */
+  const [successNotificationOpen, setSuccessNotificationOpen] = useState(false);
+  const [errorNotificationOpen, setErrorNotificationOpen] = useState(false);
+
+  /** React hook form */
+  const {
+    register,
+    handleSubmit,
+    watch,
+    getValues,
+    setValue,
+    control,
+    formState: { errors },
+  } = useForm<FormValues>(
+    eventDetails
+      ? {
+          defaultValues: {
+            startDate: eventDetails.startDate,
+            endDate: eventDetails.endDate,
+            startTime: eventDetails.startTime,
+            endTime: eventDetails.endTime,
+          },
+        }
+      : {}
+  );
+
+  /** Handles form errors for time and date validation */
+  const [errorMessage, setErrorMessage] = useState<string | null>(null);
+  const [successMessage, setSuccessMessage] = useState<string | null>(null);
+  const timeAndDateValidation = () => {
+    const { startTime, startDate, endTime, endDate } = getValues();
+    const startDateTime = convertToISO(startTime, startDate);
+    const endDateTime = convertToISO(endTime, endDate);
+    if (new Date(startDateTime) >= new Date(endDateTime)) {
+      setErrorNotificationOpen(true);
+      setErrorMessage(
+        "End Date and Time must be later than Start Date and Time"
+      );
+      return false;
+    } else {
+      setErrorMessage(null);
+    }
+    return true;
+  };
+
+  const back = () => {
+    router.push("/events/view");
+  };
+
+  /** Tanstack mutation for creating a new event */
+  const { mutateAsync, isPending, isError, isSuccess } = useMutation({
+    mutationFn: async (formData: FormValues) => {
+      // const eventid = router.query.eventid as string;
+      const { data } = await api.get(`/events/${eventid}`);
+      const { startDate, endDate, startTime, endTime } = formData;
+      const startDateTime = convertToISO(startTime, startDate);
+      const endDateTime = convertToISO(endTime, endDate);
+      const userid = await fetchUserIdFromDatabase(user?.email as string);
+      const { response } = await api.post("/events", {
+        userID: `${userid}`,
+        event: {
+          name: `${data["data"].name}`,
+          location: `${data["data"].location}`,
+          description: `${data["data"].description}`,
+          startDate: new Date(startDateTime),
+          endDate: new Date(endDateTime),
+          capacity: +data["data"].capacity,
+          mode: `${data["data"].mode}`,
+        },
+      });
```

Comment:
kindly include the `imageURL` if it exists so that the new event can have the same image